src/core/src/package_managers/ZypperPackageManager.py

```diff
@@ -79,6 +85,68 @@ def invoke_package_manager(self, command):
             self.set_package_manager_setting(Constants.PACKAGE_MGR_SETTING_REPEAT_PATCH_OPERATION, True)
         return out
 
+    def get_process_tree_from_pid_in_output(self, message):
+        """ When given a string containing a pid, returns a multi-line string with a list of parent/child processes for that pid.
```

Comment:
Change the description, to what the function does given the input. Something similar to:

""" Fetches pid from the error message, by searching for the text 'pid' and returns the process tree with all details. """

Remove this statement: "See pattern matching inside the function for details on specific strings that are accepted."
as we are calling this function for all errored instances in zypper. Instead, since we are only searching for the text 'pid' in the input, above statement should suffice

Reply:
Thanks, will do!